### ISSUE TITLE
Improve results

### ISSUE THREAD

**Issue by GalAvineri** [GalAvineri/MelMedic#1]
The current best model produced only ever predicts 'Benign' on the test set.

**Comment by GalAvineri** [comment]
The current best model can now predict 'malignant' on the test set, perhaps due to the better shuffling.
(d5531fb25f757adfd5580ca9346c8d85e9889db4)

But there are still problems:
1. The precision and recall metrics of the malignant class are always 0 at validation, which means that during validation the model never predicts 'malignant'
2. The test precision and recall metrics of the malignant class are much lower than during training, and are very close to 0.
i.e the training metrics were about 0.5 and the test metrics were about 0.03

**Comment by GalAvineri** [comment]
Further investigation: The training loss is decreases monotonically (not by much though)
but the validation loss isn't decreasing at all after the first epoch.
